File: c#/3447.cs
using System;

class Program
{
    static void Main()
    {
        string line;

        while ((line = Console.ReadLine()) != null)
        {
            while (line.Contains("BUG"))
            {
                line = line.Replace("BUG", "");
            }

            Console.WriteLine(line);
        }
    }
}

File: rust/src/03/3447.rs
use std::io;

fn main() {
    let buf = io::read_to_string(io::stdin()).unwrap();
    const BUG: &str = "BUG";

    for mut input in buf.lines().map(str::to_owned) {
        while input.contains(BUG) {
            input = input.replace(BUG, "");
        }

        println!("{input}");
    }
}
